pkg/quota/image/registry.go

```diff
@@ -8,18 +8,19 @@ import (
 	"k8s.io/kubernetes/pkg/quota"
 	"k8s.io/kubernetes/pkg/quota/generic"
 
+	imagev1 "github.com/openshift/api/image/v1"
+	imagev1typedclient "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
+	imagev1informer "github.com/openshift/client-go/image/informers/externalversions/image/v1"
 	"github.com/openshift/origin/pkg/api/legacy"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
-	imageinternalversion "github.com/openshift/origin/pkg/image/generated/informers/internalversion/image/internalversion"
-	imageclient "github.com/openshift/origin/pkg/image/generated/internalclientset/typed/image/internalversion"
 )
 
 var legacyObjectCountAliases = map[schema.GroupVersionResource]kapi.ResourceName{
-	imageapi.GroupVersion.WithResource("imagestreams"): imageapi.ResourceImageStreams,
+	imagev1.GroupVersion.WithResource("imagestreams"): imageapi.ResourceImageStreams,
```

Comment:
@deads2k @mfojtik this and a similar change bellow was missing causing quota tests to fail.